src/commands/messageContent/AutoReact.ts

```diff
@@ -0,0 +1,39 @@
+import { Message } from "discord.js";
+
+import { replaceMentions } from "../../utils/helpers";
+import { discordBot } from "../../server";
+
+const reactionMap = new Map<string | RegExp, string>([
+    [/hodor/i, "door"],
+    [/bisbopt/i, "bisbopt"],
+    [/duck/i, "duck"],
+    [/69/i, "nice"],
+    [/420/i, "weed"],
+    [/puya[1ilӏ]{1,2}up/i, "downvote"],
+    [/bruh/i, "bruh"]
+]);
+
+export default {
+    trigger: /(hodor|bisbopt|duck|69|420|puya[1ilӏ]{1,2}up|bruh)/i,
```

Comment:
Can this be generated from reactionMap so that we don't have to write it twice?

Reply:
Updated to generate from reactionMap.